src/main/java/edu/cornell/kfs/sys/xmladapters/KualiDecimalXmlAdapter.java

```diff
@@ -0,0 +1,22 @@
+package edu.cornell.kfs.sys.xmladapters;
+
+import java.math.BigDecimal;
+
+import javax.xml.bind.annotation.adapters.XmlAdapter;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+
+public class KualiDecimalXmlAdapter extends XmlAdapter<String, KualiDecimal> {
+
+    @Override
+    public KualiDecimal unmarshal(String v) throws Exception {
+        BigDecimal doubleValue = new BigDecimal(v);
+        return new KualiDecimal(doubleValue);
```

Comment:
KualiDecimal has a constructor that accepts a String as input, so you don't need to pass in an explicit BigDecimal unless you're running into problems with the other constructor.